README.md

```diff
@@ -1,21 +1,36 @@
 # Welcome to DuckDuckHack!
 
-![](http://docs.duckduckhack.com/assets/hack search engine.png)
+We're a community dedicated to improving search results with open source, "Instant Answers", that solve searches in few or zero clicks.
 
-We're a community dedicated to improving search results with open source, "Instant Answers." Instant Answers use the Web's best APIs and data to solve searches in few or zero clicks. Together and openly, we can create the best search engine for every type of search.
 
-**The DuckDuckHack community and DuckDuckGo staff are working together on a mission to create the best search engine for programmers. To fully focus there, we are only accepting Pull Requests and Issues related to the [Programming Mission](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).**
+### The Programming Mission
+The current focus of DuckDuckHack is to create the best search engine for programmers. For now, we are **only accepting Pull Requests and Issues related to the Programming Mission**.
 
-[**Join the Programming Mission!**](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53)  
-*Your help and expertise can have a huge impact on improving the search experience for developers and DuckDuckGo users around the world!*
 
-More Resources:   
-*[Full documentation](http://docs.duckduckhack.com)*  
-*[Instant Answers in Production](https://duck.co/ia)*  
-*[Instant Answers in Beta](https://beta.duckduckgo.com/?q=test&t=hc&ia=iatesting&iax=1)*  
+## Get Started
```

Comment:
While I agree about reducing duplication, I don't love the idea of hiding the most important information in another document. It makes the README considerably less useful. 

Arguably, we don't even need the CONTRIBUTING.md if there's nothing else useful to put in there. Instead we could link to the anchor on the README. If we want to keep the CONTRIBUTING.md, I think it should be more technical and detailed than this short section in the README. In my opinion, the "How to contribute" section is really the most important part as it's the call-to-action, so I don't want to hide it behind another click.